tests/topotests/bgp_color_extcommunities/r1/bgpd.conf

```diff
@@ -0,0 +1,21 @@
+!
+router bgp 65001
+ bgp router-id 192.168.1.1
+ no bgp ebgp-requires-policy
+ no bgp network import-check
+ neighbor 192.168.1.2 remote-as external
+
```

Comment:
`address-family ipv4 unicast` please instead of an empty line

Reply:
taken into account